config/routes.rb

```diff
@@ -35,6 +35,9 @@
     post 'create_enrollment' => 'teams#create_enrollment'
     resources :users, only: [:create]
     resources :api_tokens, only: [:create, :destroy]
+    member do
+      post :unarchiving
```

Comment:
Why not `unarchive`? All other actions are in the infinitive form.

Reply:
Looks better! Thanks